### PR TITLE
feat(trace.py) get trace url

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

**Review by adavidai — DISMISSED** [review]

File: epsagon/trace.py [hunk 1]
```diff
@@ -1162,6 +1161,26 @@ def _send_traces(self):
...
                 print('trace:', trace)
 
+    def get_trace_url(self):
+        """
+        Return the trace URL based on the runner ID.
+        Handling special case for Lambda functions.
+        :return: string of the trace URL.
+        """
+        trace_url = TRACE_URL_PREFIX.format(
+            id=self.runner.resource['metadata'].get('trace_id', ''),
```
Comment (adavidai) [inline epsagon/trace.py:1171]: Maybe verify there's a runner, and it has metadata? (So this will not crash in "edge cases")

**Review by adavidai — DISMISSED** [review]
See a minor comment below

File: epsagon/trace.py (hunk 1 above)
Comment (ranrib) [reply epsagon/trace.py:1178]: Fixed and added a test 👍

**Review by adavidai — APPROVED** [review]
